hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1690,14 +1690,10 @@ public boolean getAclsEnabled() {
   }
 
   /**
-   * Changes the owner of a volume.
-   *
-   * @param volume - Name of the volume.
-   * @param owner - Name of the owner.
-   * @throws IOException
+   * {@inheritDoc}
    */
   @Override
-  public void setOwner(String volume, String owner) throws IOException {
+  public boolean setOwner(String volume, String owner) throws IOException {
```

Comment:
Just curious if this ever return false?

Reply:
> Just curious if this ever return false?

Yes. It returns false when the given user is already the owner of the volume: https://github.com/apache/hadoop-ozone/pull/876/commits/4613bd0724ee0be97651aaa7364f3921436b9183#diff-990a43aeb92fe45392bdab4484ec488bR150

Reply:
Correction: Here is just to follow the change in `OzoneManagerServerProtocol`.